src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -14,27 +16,102 @@ namespace System.Net.Security.Tests
 {
     public class SslStreamStreamToStreamTest
     {
+        private readonly byte[] sampleMsg = Encoding.UTF8.GetBytes("Sample Test Message");
+        private const int TaskWaitTime = 15; // seconds
+
+        #region Tests
```

Comment:
Regions in my opinion are another code smell that shows that there's just too much in a single file. Since this grew bigger, we now need to extract the two classes below in their own files for clarity.